breeze/models/mhwp.py

```diff
@@ -32,7 +32,9 @@ def __init__(
             is_disabled=is_disabled,
         )
         self.__appointments = appointments if appointments is not None else []
-        self.__assigned_patients = assigned_patients if assigned_patients is not None else []
+        self.__assigned_patients = (
+            assigned_patients if assigned_patients is not None else []
```

Comment:
just out of curiosity, when will be the case that assigened_patients will be a tuple of list? 
if assigned_patients is always to be a list, then we can enforce it to be a list type like:
`self.__assigned_patients = list(assigned_patients) if assigned_patients is not None else []`